usr/lib/systemd/system/console-login-helper-messages-issuegen.service

```
@@ -17,6 +21,11 @@ Type=oneshot
 # effect).
 RemainAfterExit=no
 ExecStart=/usr/libexec/console-login-helper-messages/issuegen
+# See https://github.com/coreos/console-login-helper-messages/issues/87
+# We want to avoid service start limits and throttle the rate at
+# which we regenerate.  This ensures the service starts at most once
+# per second.
+ExecStartPre=/bin/sh -c 'sleep 1'
```

```suggestion
ExecStartPre=sleep 1
```

Looks a little cleaner. Also WDYT about moving the `ExecStartPre` before the `ExecStart` (it helps when thinking about the unit to see order the entries in the order the operations are going to run in).

---

Looks a lot cleaner :D 
For my knowledge though, when should we execute scripts like this in a separate process?

---

Using `sh -c` is useful if e.g. you need shell expansion or want to have some basic conditional logic, etc... But in general if it's just calling out to one program, it's better to call out to that one specifically. That removes one layer from the hierarchy and makes monitoring and logging from the service nicer.